### PR TITLE
Review comments

### PR DESCRIPTION
**Pull request by tschmidtb51**
Co-authored-by: @mprpic 

- addresses review comments of https://github.com/oasis-tcs/csaf/pull/450
- change links to use https
- update JSON schema links to draft 2020-12
- harmonize links: always use rfc-editor.org
- improve wording

### CODE REVIEW

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 1]
```diff
@@ -313,8 +313,8 @@ _Common Weakness Enumeration (CWE) – A Community-Developed List of Software We
 ###### [CYCLONEDX13]
 _CycloneDX Software Bill-of-Material Specification JSON schema version 1.3_, cyclonedx.org, May 2021, https://github.com/CycloneDX/specification/blob/1.3/schema/bom-1.3.schema.json.
 ###### [DCMI11]
-_DCMI Metadata Terms v1.1_, Dublin Core Metadata Initiative, DCMI Rec., June 14, 2012, http://dublincore.org/documents/2012/06/14/dcmi-terms/.
-Latest version available at http://dublincore.org/documents/dcmi-terms/.
+_DCMI Metadata Terms v1.1_, Dublin Core Metadata Initiative, DCMI Rec., June 14, 2012, https://dublincore.org/documents/2012/06/14/dcmi-terms/.
```
Comment (tschmidtb51) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:316]: @sthagen Is that still useful? Should we update that to the latest version?

**Review by tschmidtb51 — COMMENTED** [review]
@sthagen Should we also update  ISO8601? We link to https://www.iso.org/standard/40874.html which has been superseeded by 8601-1 and 8601-2..

**Review by sthagen — APPROVED** [review]

**Comment by sthagen** [comment]
I had to rerun the checks (connectivity problems in azure cloud?)

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 1 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: I had the same problem today...

**Comment by tschmidtb51** [comment]
> @sthagen Should we also update ISO8601? We link to https://www.iso.org/standard/40874.html which has been superseeded by 8601-1 and 8601-2..

What about the ISO8601?

**Comment by sthagen** [comment]
I would keep the ISO8601 as is. I am not in the mood of buying this edition also and I am sure they did not change anything important that would impact the parts we use.

**Pull request merged**